src/SelectPanel/SelectPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useCallback, useMemo} from 'react'
+
+import {FilteredActionList, FilteredActionListProps} from '../FilteredActionList'
+import {OverlayProps} from '../Overlay'
+import {ItemInput} from '../ActionList/List'
+import {FocusZoneSettings} from '../behaviors/focusZone'
+import {DropdownButton, DropdownButtonProps} from '../DropdownMenu'
+import {ItemProps} from '../ActionList'
+import {AnchoredOverlay} from '../AnchoredOverlay'
+import Flex from '../Flex'
+
+export interface SelectPanelProps extends Omit<FilteredActionListProps, 'onChange'> {
+  open: boolean
+  onOpen: (gesture: 'anchor-click' | 'anchor-key-press') => unknown
```

Comment:
```suggestion
  onOpen: AnchoredOverlay['onOpen']
```
Suggestion: Since this is passed through to `AnchoredOverlay`, consider referencing its [`onOpen` type](https://github.com/primer/components/blob/84c7e771f8ac9bb707fbb977658e9e40620bc20c/src/AnchoredOverlay/AnchoredOverlay.tsx#L21-L24).

Alternative suggestion: In addition to `extends`’ing `Omit<FilteredActionListProps, 'onChange'>`, also `extends` from `Pick<AnchoredOverlay, 'onOpen' | /*…*/ >`